src/Table/TableHead.js

```diff
@@ -84,11 +85,18 @@ class TableHead extends Component {
       }
     )
 
+    let thWidth
```

Comment:
Creio que aqui seja mais interessante utilizar a tag `style` como base, para que caso seja necessária a adição de alguma otra configuração de estilos possa ser feita de maneira simples.
```suggestion
    const thStyle = {}
```

Reply:
Eu alterei, mas pq essa abordagem é melhor? Ou é apenas uma abordagem diferente? Melhor deixar como `const` do que `let` nesse caso? Vejo que o ganho aqui é que não há a alteração de valor. da variável. Seria isso?